src/components/Events/EventCard.js

```diff
@@ -70,54 +69,59 @@ const AlignRightCol = styled(Col)`
 
 const EventCard = ({ event }) => {
   const {
-    type,
     date,
-    eventName,
-    eventLocation,
+    addressLine3,
+    city,
     startTime,
     endTime,
-    attendees = 0,
-    link
+    type,
+    eventTitle,
+    attendees,
+    linkToEvent
   } = event
 
+  const formattedStartTime = format(new Date(startTime), 'ha')
+  const formattedEndTime = format(new Date(endTime), 'ha')
+
+  const eventInfo = `${
+    addressLine3 ? `${addressLine3}, ` : ''
+  } ${city} • ${formattedStartTime} - ${formattedEndTime}${
```

Comment:
this will make the string start with an empty space if there's no address defined. Space character is alread after the comma, so no need for an extra space before `${city}` 😉

Reply:
ah yes, nice spot. I really dislike string interpolation...